typings/type.ts

```diff
@@ -150,5 +150,23 @@ ComponentNext.from(
   10
 )
 
-// $ExpectType ComponentNext<{ iState: undefined; oState: undefined; oView: void; }>
+// $ExpectType ComponentNext<{ iState: undefined; oView: void; }>
 ComponentNext.empty
+
+// $ExpectType { b: string; a: string; } | { c: string; a: string; }
+$(
+  ComponentNext.lift({a: ''})
+    .matchR('action1', (value, state) => ({...state, b: ''}))
+    .matchR('action2', (value, state) => ({...state, c: ''}))
+).oState
+
+// $ExpectType { a: string; } | { b: string; a: string; } | { c: string; a: string; }
```

Comment:
maybe I misunderstood, but wasn't `oView` the type of what is returned by `render`?

Here it looks like it's the type of the state that the render function accepts.

Reply:
render is returning state here